test/unit/range/views/view_minimiser_test.cpp

```diff
@@ -26,11 +28,24 @@ using result_t = std::vector<size_t>;
 using iterator_type = std::ranges::iterator_t< decltype(std::declval<seqan3::dna4_vector&>()
                                                         | seqan3::views::kmer_hash(seqan3::ungapped{4})
                                                         | seqan3::views::minimiser(5))>;
+using two_ranges_iterator_type = std::ranges::iterator_t< decltype(std::declval<seqan3::dna4_vector&>()
+                                                                   | seqan3::views::kmer_hash(seqan3::ungapped{4})
+                                                                   | seqan3::views::minimiser(5,
+                                                                       std::declval<seqan3::dna4_vector&>()
+                                                                       | seqan3::views::complement | std::views::reverse
+                                                                       | seqan3::views::kmer_hash(seqan3::ungapped{4})
+                                                                       | std::views::reverse))>;
 
 static constexpr auto kmer_view = seqan3::views::kmer_hash(seqan3::ungapped{4});
+static constexpr auto rev_kmer_view = seqan3::views::complement | std::views::reverse
```

Comment:
Please mark them all as inline.
Also you could safe some writing above if you would declare them before the type definitions and use them inside of the decltype expressions.